feature-libs/product-configurator/rulebased/core/model/configurator.model.ts

```diff
@@ -105,6 +105,7 @@ export namespace Configurator {
     hideBasePriceAndSelectedOptions?: boolean;
     immediateConflictResolution?: boolean;
     newConfiguration?: boolean;
+    isAsyncPricing?: boolean;
```

Comment:
Actually correct would: 'isPricingEnabledAndIsAsyncPricingAndIsPerformanceFeatureToggleOn'.
However isAsyncPricing implies that isPricingEnabled and the feature toggle will go away, so i came up with this name.
PS: isAsyncPricing === isCPS (CPQ is sync pricing)